_includes/consdata-offers.html

```diff
@@ -0,0 +1,17 @@
+<div class="sidebar-container" id="recommended-posts-sidebar">
+    <div class="sidebar-title">
+        {{ site.data.translations['joinUs'][page.lang] }}
+    </div>
+    <div class="sidebar-latest-posts-container">
+        <ul class="sidebar-posts">
+            <li class="sidebar-post no-breaking-word">
+                <a href="https://consdata.com/pl/kariera/dlaczego-consdata" lang="pl" target="_blank">
```

Comment:
lang=pl?

Reply:
Racja, jeśli zależnie od tego co Adam niżej odpowie to to usunę (jeśli zostawimy ang), albo zostawię, jeśli będzie tylko po polsku

Reply:
Usunięte